### PR TITLE
Fixed incorrect optional/nullable modeling for Webhook.ApplicationID

### PR DESCRIPTION
**Pull request by JakenVeina**

### CODE REVIEW

**Comment by Cisien** [comment]
Can confirm this works when I bypass the API wrapper and make the http call directly with the DiscordHttpClient

File: Tests/Remora.Discord.Tests/Samples/Objects/WEBHOOK/2021-07-10.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
```
Comment (Nihlus) [inline Tests/Remora.Discord.Tests/Samples/Objects/WEBHOOK/2021-07-10.json:1]: Run this file through the scrubber first.

**Review by Nihlus — CHANGES_REQUESTED** [review]

File: Tests/Remora.Discord.Tests/Samples/Objects/WEBHOOK/2021-07-10.json (hunk 1 above)
Comment (JakenVeina) [reply Tests/Remora.Discord.Tests/Samples/Objects/WEBHOOK/2021-07-10.json:1]: Ah, right. I forgot about that. I asked Cisien first if he was cool with the sample being saved, but that makes more sense.

Comment (JakenVeina) [reply Tests/Remora.Discord.Tests/Samples/Objects/WEBHOOK/2021-07-10.json:1]: Force-pushed to eliminate the unscrubbed file from history.

**Pull request merged**